signing/src/main/java/tech/pegasys/web3signer/signing/config/SignerLoader.java

```diff
@@ -49,6 +49,11 @@ public class SignerLoader {
   private static final Logger LOG = LogManager.getLogger();
   private static final long FILES_PROCESSED_TO_REPORT = 10;
   private static final int MAX_FORK_JOIN_THREADS = 5;
+  // by default, we always want to process hashicorp connections in parallel. For corner cases when
+  // dealing with
+  // 30,000+ files, we can enable sequential processing using system property at command line.
+  private static final boolean handleHashicorpConfigurationSequential =
```

Comment:
Would rather this be a command line option to be consistent with our other configuration. We can make it a hidden one with the X prefix.